projects/angular/src/forms/datepicker/datepicker-action-buttons.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2016-2024 Broadcom. All Rights Reserved.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Component, Directive, HostListener } from '@angular/core';
+
+import { ClrPopoverToggleService } from '../../utils/popover/providers/popover-toggle.service';
+import { DateFormControlService } from './providers/date-form-control.service';
+import { DateNavigationService } from './providers/date-navigation.service';
+
+@Component({
+  selector: 'clr-datepicker-actions',
+  template: ` <ng-content></ng-content> `,
+  host: {
+    '[class.datepicker-actions]': 'true',
+  },
+})
+export class ClrDatepickerActions {}
+
+@Directive({
+  selector: '[clrDatepickerApplyAction]',
```

Comment:
```suggestion
  selector: 'button[clrDatepickerApplyAction]',
```

I would make this selector specific to `button` elements.

Reply:
This is a moot point if you remove this directive as I suggested below.